test/integration/machine-learning.spec.ts

```diff
@@ -324,6 +324,124 @@ describe('admin.machineLearning', () => {
     });
   });
 
+  describe('listModels()', () => {
```

Comment:
I'd recommend organizing this test as follows to reduce the rpc overhead of creating/deleting many temporary models.

```
describe('listModels()', () => {
  before(() => {
     // create 3-4 temp models
  });

  // individual it() cases each reusing the models created in before()
 it('resolves with a list of models', () => {
    return admin.machineLearning().listModels(...);
 });
});
```

This has the added benefit that each test case only calls `listModels()` and none of the setup/create methods.

Reply:
done